templates/api/knowledge-panels/health/ingredients/ingredient.tt.json

```diff
@@ -0,0 +1,37 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "size": "small",
+    [% IF panel.ingredient_description OR panel.wikipedia_abstract %]
+    "expanded": false,
+    [% ELSE %]
+    "expanded": false,
```

Comment:
This is strange !

Shouldn't the second expanded be true ?

Reply:
It's temporary, in fact what we really want is to make the first one (no description) not expandable. But currently the non expandable panels are displayed in a very different way (at least on the web, I haven't checked mobile). So we will need a better way.